srcpkgs/fontmanager/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'fontmanager'
 pkgname=fontmanager
-version=0.8.6
+version=0.8.7
 revision=1
 wrksrc="font-manager-${version}"
 build_helper=gir
 build_style=meson
 hostmakedepends="pkg-config yelp-tools vala-devel glib-devel gettext"
 makedepends="json-glib-devel sqlite-devel gtk+3-devel fontconfig-devel
  libsoup-devel webkit2gtk-devel"
-short_desc="Simple font management for GTK+ desktop environments"
+short_desc="Font-manager is a simple font management tool for GTK+ environments"
```

Comment:
Package name shouldn/t be included in desc.

Reply:
I do this to make searching for this package simpler. There is a little unfortunate inconsistency between the Voidlinux package name and the actual package name (Voidlinux uses `fontmnager` and practically every other package manager uses the official name `font-manager`). This change change costs basically nothing and it makes font-manager searchable via `xbpq-query -Rs font-manager`. If you do not agree with me, I will remove this.

Reply:
Uh, okay.